components/match2/wikis/valorant/big_match.lua

```diff
@@ -22,6 +22,10 @@ local CustomMatchGroupInput = Lua.import('Module:MatchGroup/Input/Custom', {requ
 
 local BigMatch = Class.new()
 
+local _ROUNDS_PER_HALF = 12
+local _COLOR_FIRST_TEAM = '#B12A2A'
+local _COLOR_SECOND_TEAM = '#1E7D7D'
```

Comment:
those are just for the charts, right?
if not we should try to use css instead, for chart iirc that wasn't possible

Reply:
Charts